src/core/layout/scrollable-list/scrollable-list-buttons.js

```diff
@@ -22,13 +24,26 @@ const createGelButton = (scene, item) => {
         group: scene.scene.key,
         id,
         key: config.assetKeys.itemBackground,
-        scene: config.assetKeys.prefix,
+        scene: config.assetPrefix,
```

Comment:
guess scene isnt the actual scene :D

also bit confused as why theres scene.key and scene.assetPrefix being used here